courses/free-courses-en.md

```diff
@@ -966,6 +966,7 @@
 * [Angular Crash Course 2021](https://www.youtube.com/watch?v=3dHNOWTI7H8) - Brad Traversy, Traversy Media
 * [Angular Fast Crash Course](https://www.udemy.com/angular-fast-crash-course/) - Edwin Diaz, Coding Faculty Solutions (Udemy)
 * [Angular for Beginners](https://www.udemy.com/course/angular-for-beginners-course/) - Angular University (Udemy)
+* [Angular in 100 Seconds](https://youtu.be/Ata9cSC2WpM?si=R1FfCRhBG7FJvSvk) -  Fireship
```

Comment:
we don't accept shortened links
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Its not a playlist, but just one video teaching Angular @eshellman

Reply:
youtu.be is a shortened link.